examples/kafka/kafka-ephemeral-oauth-single-keycloak-authz.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    version: 2.7.0
+    replicas: 1
+    listeners:
+      - name: plain
+        port: 9092
+        type: internal
+        tls: false
```

Comment:
Should we remove this given it is security example?

Reply:
Actually we could yes.